apps/frontend/src/pages/profile/profile.tsx

```diff
@@ -41,6 +44,18 @@ const Profile: React.FC = () => {
 		[dispatch, user],
 	);
 
+	const handleUpdatePasswordSubmit = useCallback(
+		(payload: Omit<UserUpdatePasswordRequestDto, "email">): void => {
+			void dispatch(
+				authActions.updatePassword({
+					email: user?.email as string,
```

Comment:
Why do we need to send email? We can get it from token in Backend

Reply:
Do you mean I should find the user's email using their Id? because the jwt token payload does not contain the email directly

![signup-func](https://github.com/user-attachments/assets/fc054b6b-1dc2-470f-b6c6-d09e75a32f3b)

Reply:
We have authorization plugin, which injects the current user to the request. Use it